app/views/stories/_sign_in_invitation.html.erb

```diff
@@ -1,24 +1,24 @@
-<div class="single-article single-article-small-pic feed-cta" id="in-feed-cta">
-  <div class="cta-container" id="cta-content">
-    <img class="secondary-logo" src="<%= cloudinary(safe_logo_url(SiteConfig.secondary_logo_url), 190) %>"
-         alt="Secondary <%= community_name %> logo" />
-
-    <h2>
-      <a href="/"><%= community_name %></a> is a community of <br />
-      <%= number_with_delimiter User.registered.estimated_count %> amazing <%= community_members_label %>
-    </h2>
-
-    <h3>
-      Create your profile to customize your experience and get involved.
-    </h3>
-
-    <div class="button-container">
-      <a href="/enter" class="cta cta-button" aria-label="Sign up.">
-        Join Now
-      </a>
-      <div class="feed-cta-sub">
-        <em>We require social login to prevent abuse.</em>
-      </div>
+<div class="authentication-feed__card" id="in-feed-cta">
+  <div class="authentication-feed__container">
+    <img class="authentication-feed__logo" src="<%= cloudinary(safe_logo_url(SiteConfig.secondary_logo_url), 190) %>"
+          alt="Secondary <%= community_name %> logo" />
+    <div class="authentication-feed__content">
+      <h2 class="authentication-feed__title">
+<a href="/"><%= community_name %></a> is a community of <br />
+        <%= number_with_delimiter User.registered.estimated_count %> amazing <%= community_members_label %>
+      </h2>
+      <p class="authentication-feed__description">
+        Log in to customize your experience and get involved.
+      </p>
     </div>
   </div>
+
+  <div class="authentication-feed__actions">
+    <a href="/enter" class="crayons-btn" aria-label="Create new account">
+      Create new account
+    </a>
+    <a href="/enter" class="crayons-btn crayons-btn--ghost-brand" aria-label="Log in">
+      Log in
+    </a>
```

Comment:
Why two separate buttons taking to the same place?

Reply:
This is the fix for now to make sure that these actions address people's different intentions, but we're doing this in a phased way, and eventually this will lead people to a split experience.